certified-connectors/CardPlatform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,691 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "CardPlatform",
```

Comment:
@st0014 Please add space between `Card` and `Platform`

Reply:
The name of the product is "CardPlatform" - one word.

I have updated title to "CardPlatform Adaptive Cards"